app/src/main/java/org/projectbuendia/client/json/JsonOrdersResponse.java

```diff
@@ -0,0 +1,13 @@
+package org.projectbuendia.client.json;
+
+/**
+ * JSON representation of a set of patients returned by the server.
+ * <p>
+ * TODO: Generify this class into e.g. an IncrementalFetchResponse<JsonEncounter>
+ */
+public class JsonOrdersResponse {
+    public JsonOrder[] results;
+    // TODO(capnfabs): Rename this to syncToken.
+    /** In ISO 8601 date format. */
+    public String snapshotTime;
```

Comment:
Jax-RS @produces and @consumes are so much more fun. You would use DateTime here.